pipeline/extract/Covid_cases_deaths_data.sql

```diff
@@ -0,0 +1,25 @@
+select
+    date_extract_y(lab_report_date) as year,
+    date_extract_woy(lab_report_date) as week,
+    sum(cases_age_0_17),
+    sum(cases_age_18_29),
+    sum(cases_age_30_39),
+    sum(cases_age_40_49),
+    sum(cases_age_50_59),
+    sum(cases_age_60_69),
+    sum(cases_age_70_79),
+    sum(cases_age_80_),
+    sum(deaths_0_17_yrs),
+    sum(deaths_18_29_yrs),
+    sum(deaths_30_39_yrs),
+    sum(deaths_40_49_yrs),
+    sum(deaths_50_59_yrs),
+    sum(deaths_60_69_yrs),
+    sum(deaths_70_79_yrs),
+    sum(deaths_80_yrs),
+    Min(lab_report_date) as week_start,
+    Max(lab_report_date) as week_end
```

Comment:
Can you add a where clause to restrict to only rows less than or equal to a certain lab report date? The lab report date for that where clause should be whatever is the current most recent report date for this dataset, which you can check on the data portal: https://data.cityofchicago.org/Health-Human-Services/COVID-19-Daily-Cases-Deaths-and-Hospitalizations/naz8-j4nc

The city updates this dataset very frequently, so we want to control when we get updates. For more information on why this is important and can cause problems, see #5